Automattic/wp-calypso
# A4A: Fixing various flows in the sites preview pane Activity panel

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/137

## Proposed Changes

* This PR adds an upsell on the Activity panel where relevant, linking to the marketplace/products page. It also adds a copy site button.
* Note - the wording has now changed to 'Activity log' - do we want that to change back to Activity, and match the size (plus the subtitle size) as before?


What doesn't work yet:

* The copy site button functionality - it works for Atomic sites but not Jetpack sites (yet - functionality to come later when redirects added).
* I've also left the links in the Activity log themselves on the assumption that they will all have redirections - coming soon.
* Similarly, the 'remove all filters' button does not work (it redirects you to a non existent Activity Log page), but that should also be fixed when redirects are added.


## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally and then visit `http://agencies.localhost:3000/sites`.

* Open the preview pane for any site and visit the Activity panel. The above changes should be visible, depending on the site type and also upgrade level.


Upsell box:

<img width="1149" alt="Screenshot 2024-04-05 at 12 07 58" src="https://github.com/Automattic/wp-calypso/assets/16754605/6cc354e5-4136-4db4-98ac-82b88bfccfb1">



Copy site button:

<img width="1120" alt="Screenshot 2024-04-05 at 12 08 06" src="https://github.com/Automattic/wp-calypso/assets/16754605/11fcbdf5-22e1-409c-9808-0f6a6039485f">


## Pre-merge Checklist
